src/explorer/SubscriptionTreeItem.ts

```diff
@@ -103,19 +105,36 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         if (!advancedCreation) {
             // this should always be set when in the basic creation scenario
             const location: Location = nonNullProp(wizardContext, 'location');
-            wizardContext.newResourceGroupName = `appsvc_rg_${wizardContext.newSiteOS}_${location.name}`;
-            wizardContext.newPlanName = `appsvc_asp_${wizardContext.newSiteOS}_${location.name}`;
+            wizardContext.newResourceGroupName = `appsvc_${wizardContext.newSiteOS}_${location.name}`;
+            wizardContext.newPlanName = wizardContext.newResourceGroupName;
```

Comment:
Move these few lines inside `setDefaultRgAndPlanName` (the lines that are setting the first defaults)

Reply:
😂